src/main/java/io/phasetwo/keycloak/magic/MagicLink.java

```diff
@@ -93,12 +93,12 @@ public static UserModel getOrCreate(
   }
 
   public static MagicLinkActionToken createActionToken(
-      UserModel user, String clientId, String redirectUri, OptionalInt validity) {
+      UserModel user, String clientId, String redirectUri, String scopes, OptionalInt validity) {
```

Comment:
Since this method was public, can you maintain the old method, and call the new method with a default for `scopes`?

Reply:
Will do.

Reply:
Thanks. I'll merge once you update the PR.

Reply:
@xgp Done.